services/ui-src/src/verbiage/pages/mcpar/mcpar-successful-submit.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  intro: {
+    header: "Successfully Submitted",
+    infoHeader: "Thank you",
+    additionalInfoHeader: "What happens now?",
+    additionalInfo:
+      "No further action is needed at this point. CMS will reach out if in the case they have any questions.",
+  },
+};
```

Comment:
Let's combine this into the other `review-and-submit` verbiage object so we just have the one object for the page.